codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/UnionGenerator.kt

```diff
@@ -110,10 +132,45 @@ class UnionGenerator(
         }
     }
 
+    private fun renderDebugImplForUnion() {
```

Comment:
Consider renaming to `renderFullyRedactedDebugImpl`

Reply:
Addressed in 0f8102b